CMakeLists.txt

```diff
@@ -76,13 +78,13 @@ if (WIN32)
 	   NAME "lib3MF"
 	   FILE_DESCRIPTION "Lib3MF is the reference-implementation of the 3D Manufacturing Format file standard"
 	   BUNDLE "3MF Consortium"
-	   VERSION_MAJOR ${LIB3MF_VERSION_MAJOR}
-	   VERSION_MINOR ${LIB3MF_VERSION_MINOR}
-	   VERSION_PATCH ${LIB3MF_VERSION_MICRO}
+	   VERSION_MAJOR ${PROJECT_VERSION_MAJOR}
+	   VERSION_MINOR ${PROJECT_VERSION_MINOR}
+	   VERSION_PATCH ${PROJECT_VERSION_PATCH}
 	   VERSION_REVISION ${BUILD_NUMBER}
-	   PRODUCT_VERSION_MAJOR ${LIB3MF_VERSION_MAJOR}
-	   PRODUCT_VERSION_MINOR ${LIB3MF_VERSION_MINOR}
-	   PRODUCT_VERSION_PATCH ${LIB3MF_VERSION_MICRO}
+	   LIB3MF_VERSION_MAJOR ${PROJECT_VERSION_MAJOR}
```

Comment:
To get correct version information into the Windows DLL, Release\VersionInfo\VersionInfo.in requires PRODUCT_VERSION_* instead of LIB3MF_VERSION_* :

```
	   PRODUCT_VERSION_MAJOR ${PROJECT_VERSION_MAJOR}
	   PRODUCT_VERSION_MINOR ${PROJECT_VERSION_MINOR}
	   PRODUCT_VERSION_PATCH ${PROJECT_VERSION_PATCH}
```
This will completely remove LIB3MF_VERSION_* .

Reply:
Thanks for the review, `CMakeLists.txt` is updated to use `PRODUCT_VERSION_*`